### ISSUE TITLE
Add member list ui

### ISSUE THREAD

**Issue by toger5** [toger5/TheBoard#7]
There should be a ui for showing all the members in a room/board. This is also would allow for only showing or highlighting strokes from a specific user.

**Comment by johann1525** [comment]
True. And showing a tool tip when hovering a stroke would also be useful to see who drew what.
